MeetupPlatformApi/Entities/MeetupEntity.cs

```diff
@@ -1,14 +1,22 @@
-﻿namespace MeetupPlatformApi.Entities;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.Entities;
 
 public class MeetupEntity
 {
+    [Required]
+    [Key]
```

Comment:
The same applies to the `[Key]` attribute: if field is named `Id`, then EF Core considers it a `Primary Key` by default